### ISSUE TITLE
Style Not Implementing Any Changes

### ISSUE THREAD

**Issue by ImpossiblyInvisible** [sapondanaisriwan/AdashimaaTube#43]
I've just installed Stylus, and when I use Youtube no actual changes to the layout occur.  I have the Free Download Manager, Ublock, and DuckDuckGo extensions and I'm not sure if there's any interference from them.  I am currently using Google Chrome. Is there any Youtube UI update issue that could be causing this?

![2023-03-10_16-48-37](https://user-images.githubusercontent.com/127548875/224453182-136347de-f0d3-4f69-81c9-fb0d982dcfec.png)

![2023-03-10_16-48-04](https://user-images.githubusercontent.com/127548875/224453198-66b52915-1c78-474b-a15d-3e00392a66d2.png)

**Comment by sapondanaisriwan** [comment]
First, you need to enable the "Video Appearance."
![U3r2JcACws](https://user-images.githubusercontent.com/64634605/224443489-06381e23-c624-447c-b674-37ef151f8fc7.png)

If it's still not working, send me a screenshot or a video of your screen.

**Comment by ImpossiblyInvisible** [comment]
The style started working only until dark mode kicked in, which is weird considering I didn't choose dark mode while installing. Video appearance was already enabled before this. Will try to see if it works once it's on light mode again tomorrow

**Comment by sapondanaisriwan** [comment]
btw I can't open the images that you provide

**Comment by sapondanaisriwan** [comment]
> I've just installed Stylus, and when I use Youtube no actual changes to the layout occur. I have the Free Download Manager, Ublock, and DuckDuckGo extensions and I'm not sure if there's any interference from them. I am currently using Google Chrome. Is there any Youtube UI update issue that could be causing this?
> 
> ![2023-03-10_16-48-37](https://user-images.githubusercontent.com/127548875/224453182-136347de-f0d3-4f69-81c9-fb0d982dcfec.png)
> 
> ![2023-03-10_16-48-04](https://user-images.githubusercontent.com/127548875/224453198-66b52915-1c78-474b-a15d-3e00392a66d2.png)

Did you really install AdashimaaTube?
The watch page is still new layout of YouTube in the second picture.

**Comment by ImpossiblyInvisible** [comment]
I did install it, and made sure it was checked in Stylus. The icon had a 1 on the lower right when it activated at night.

**Comment by sapondanaisriwan** [comment]
Can you take a full screenshot?

**Comment by ImpossiblyInvisible** [comment]
In a dew hours

**Comment by ImpossiblyInvisible** [comment]
> Can you take a full screenshot?

![2023-03-11_15-29-24](https://user-images.githubusercontent.com/127548875/224505566-f8920644-ca70-4d59-868f-26f934ab5728.png)

**Comment by sapondanaisriwan** [comment]
That's weird

**Comment by ImpossiblyInvisible** [comment]
UPDATE: Apparently there are some problems with this and Stylus Night Time mode. Even though light/dark mode is disabled for this style, every time I tried reinstalling it defaulted to Dark/Light even if you manually change it. Since I had Stylus on night time mode, the style only appeared around that time (because when I first installed it from "None (always applied)" to "Dark"). When I reinstalled it for Light Mode, it also didn't let me switch to None, and only worked during day time hours.

So far not much of an issue due to personally not using Stylus itself at all, but it's still worth noting
